theme/theme.inc

```diff
@@ -16,20 +16,15 @@ function islandora_internet_archive_bookreader_preprocess_islandora_internet_arc
   $colorbox_path = libraries_get_path('colorbox');
   $module_path = drupal_get_path('module', 'islandora_internet_archive_bookreader');
   $search_uri = module_exists('islandora_solr') ? url("internet_archive_bookreader_search/{$object->id}/TERM", array('absolute' => TRUE)) : NULL;
+  $backup_uris = variable_get('islandora_internet_archive_bookreader_use_backup_dsid', FALSE);
 
-  module_load_include('inc', 'islandora', 'includes/authtokens');
   $pages = array();
   foreach ($variables['pages'] as $page_pid => $info) {
-    $token = islandora_get_object_token($page_pid, $dsid, 2);
-
-    // TODO: Slap the widths/heights for pages in here, and make 'em be used in
-    // on the JS side...  Probably break out this structure generation into a
-    // separate function?
-    $pages[] = $info + array(
+    $pages[] = $info + $backup_uris ? array('pid' => $page_pid) : array(
```

Comment:
`Fatal error: Unsupported operand types in /var/www/drupal7/sites/all/modules/islandora_internet_archive_bookreader/theme/theme.inc on line 23 `

... Can probably slap parentheses around the ternary?